clover_admin.rb

```diff
@@ -194,6 +194,26 @@ def self.object_action(...)
           fail CloverError.new(400, "InvalidRequest", "invalid JSON for feature flag value")
         end
         obj.send("set_ff_#{name}", value)
+      end,
+      "set_quota" => object_action("Set Quota", "Set quota", {
+        resource_type: {
+          typecast: :str!,
+          type: "select",
+          add_blank: true,
+          options: ProjectQuota.default_quotas.keys
+        },
+        value: {typecast: :int, type: "number", placeholder: "blank to reset to default", required: nil}
```

Comment:
Seems weird to format the `value` hash different from the `resource_type` hash. Please use a single line for both, or use a separate line per key for both.